data/data-pipeline/data_pipeline/etl/score/constants.py

```diff
@@ -240,14 +240,10 @@
     + field_names.PERCENTILE_FIELD_SUFFIX: "LIF_PFS",
     field_names.LOW_MEDIAN_INCOME_AS_PERCENT_OF_AMI_FIELD
     + field_names.PERCENTILE_FIELD_SUFFIX: "LMI_PFS",
-    field_names.MEDIAN_HOUSE_VALUE_FIELD
```

Comment:
@emma-nechamkin I would love your eyes on the changed I made to this file --- are any of these surprising to you? From what I could tell, none of these vars are used by the frontend, but if there are any that you're like "wait the frontend **really** should be using that, what is going on?!" it would be helpful for @vim-usds and me to know.

Reply:
yeah so this is fine -- we only use this for lead and i think it's abstracted!